### DISCUSSION TITLE
v2 next.js 服务端渲染样式引入失败

### DISCUSSION THREAD

**Discussion by huaxiangsirenbutong** [alibaba/formily#2311] [Q&A 2.x] [answered]
求问怎么解决
![image](https://user-images.githubusercontent.com/37055241/137428963-8c841997-f916-46c0-9a70-faba46231270.png)
![image](https://user-images.githubusercontent.com/37055241/137428982-66e3de75-e22e-49ce-9464-bb19305511bc.png)

**Comment by janryWang** [comment, marked as answer]
已经修了，等发版吧

**Reply by huaxiangsirenbutong** [reply]
你好，请问什么时候发版啊 @janryWang

**Reply by janryWang** [reply]
已经发了呀

**Reply by huaxiangsirenbutong** [reply]
似乎还是不行
报错信息：
![image](https://user-images.githubusercontent.com/37055241/137674146-eeb9589b-eed0-4fa5-b1cf-6df0ab165d1c.png)

版本：
![image](https://user-images.githubusercontent.com/37055241/137674173-0ae8c4cc-1101-4762-9f87-bf9e8f8b1ac1.png)
@janryWang ，麻烦看下

**Comment by janryWang** [comment]
感觉你的配置有问题吧，构建产物里不存在array-cards/style/css这样的路径

**Reply by huaxiangsirenbutong** [reply]
没有做多余的配置，都是按照官网来的
next.config.js配置：
![image](https://user-images.githubusercontent.com/37055241/137677781-6ab650e9-cd54-4627-a87d-6521aeb8f57d.png)
报错文件：就是直接复制的你们官方的表单设计器的代码：如下
![image](https://user-images.githubusercontent.com/37055241/137678012-b9836cb8-c645-4cce-aedd-94a478232614.png)
麻烦帮忙看下 @janryWang

**Reply by huaxiangsirenbutong** [reply]
![image](https://user-images.githubusercontent.com/37055241/137681032-c1e284e8-44a9-45f1-8986-a4451b3c0655.png)
看了一下构建产物里是存在array-cards/style/css这个文件的 @janryWang 帮忙看下
